content/master/concepts/managed-resources.md

```diff
@@ -211,37 +211,111 @@ resource object is deleted from Kubernetes and the `deletionPolicy` is
  
 
 <!-- vale off -->
-### managementPolicy
+### managementPolicies
 <!-- vale on --> 
 
 {{<hint "important" >}}
-The managed resource `managementPolicy` option is an alpha feature. 
+The managed resource `managementPolicies` option is an alpha feature. 
 
-Enable the `managementPolicy` in a provider with `--enable-management-policies` 
+Enable the `managementPolicies` in a provider with `--enable-management-policies` 
 in a 
 [ControllerConfig]({{<ref "./providers#controller-configuration" >}}).
 {{< /hint >}}
 
-A `managementPolicy` determines if Crossplane can make changes to managed
-resources. The `ObserveOnly` policy imports existing external resources not 
-originally created by Crossplane.  
-This allows new managed resources to reference 
-the `ObserveOnly` resource, for example, a shared database or network. 
-The `ObserveOnly` policy can also place existing resources under the control of
+Granular `managementPolicies` determine what actions Crossplane can take on a 
+managed and its external resource. `managementPolicies` is an array of actions
+(`Observe`, `Create`, `Update`, `Delete`, `LateInitialize`, `*`), and there are
+multiple combinations that Crossplane supports based on the use case. For 
+example, setting `["Observe"]` makes Crossplane treat the resource as an
+`ObserveOnly` resource, importing the external resources not originally created
+by Crossplane. This allows new managed resources to reference the `ObserveOnly`
```

Comment:
> importing the external resources not originally created by Crossplane.

Import is not limited to `["Observe"]`. One can still import using `["*"]` as long as they provide all required parameters. Observe only just makes importing more convenient (since you don't have to provide all required params) and safer (since provider doesn't immediately take control).